database/seeders/DatabaseSeeder.php

```diff
@@ -12,11 +12,6 @@ class DatabaseSeeder extends Seeder
      */
     public function run(): void
     {
-        // \App\Models\User::factory(10)->create();
-
-        // \App\Models\User::factory()->create([
-        //     'name' => 'Test User',
-        //     'email' => 'test@example.com',
-        // ]);
+        Flight::factory(10)->create();
```

Comment:
After the changes about Factories, you can use here `FlightFactory::new()->count(10)->create();`